foundry.toml

```diff
@@ -3,22 +3,22 @@
 # See more config options at: https://github.com/gakonst/foundry/tree/master/config
 
 # The Default Profile
-[default]
+[profile.default]
```

Comment:
Seems this PR needs a rebase, since these changes were already made in #54

Reply:
Rebased!